make/manifest.mk

```diff
@@ -19,4 +19,4 @@ manifest/check/graph:
 	delorean ews check-olm-graph -d ./manifests
 
 .PHONY: manifest/check
-manifest/check: manifest/check/image_mirror_mapping manifest/check/graph
+manifest/check: manifest/check/image_mirror_mapping # manifest/check/graph
```

Comment:
suggest to remove this entirely

Reply:
actualy, let's leave as it is (but instead of uncomment, just delete the manifest/check/graph entirely but leave the top level make command)

Reply:
Done. thank you